validator/__init__.py

```diff
@@ -0,0 +1,10 @@
+import json
+from pathlib import Path
+
+def get_latest_version():
```

Comment:
Why have a function definition in the init file?

An init file is imported *every time* the path comes in an import.

So if you do:

```
import validator.something
import validator.something.else
import validator.boo
```

In all those case the __init__ of the module is run. Here it does nothing, but has a function. Why not put it in a utils.py underneath and have a clean init? Code in init is mostly used to specify what should be imported when you do import module. 

For example: https://github.com/rotki/rotki/blob/88760576331e63e2c92eb8473debcdd61639c393/rotkehlchen/chain/ethereum/modules/uniswap/__init__.py

I don't really like putting anything in there since this makes is easy to have cyclic import error and spend hour/s fixing them